### PR TITLE
SC-8413 Upgrade Node 12 to 18 LTS

### PR DESCRIPTION
**Pull request by dione-kist-jemmic**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: 'Task Completed Checker'
 description: 'An action that checks if all tasks are completed in the pull requests.'
 author: 'kentaro-m (forked by kamphaus)'
 runs:
-  using: 'node12'
+  using: 'node16'
```
Comment (christophe-kamphaus-jemmic) [inline action.yml]: Shouldn't this be?
```suggestion
  using: 'node18'
```

File: package.json [hunk 2]
```diff
@@ -25,25 +25,26 @@
   "author": "kentaro-m",
   "license": "MIT",
   "dependencies": {
-    "@actions/core": "^1.4.0",
-    "@actions/github": "^5.0.0",
-    "marked": "^2.1.3"
+    "@actions/core": "^1.10.1",
+    "@actions/github": "^5.1.1",
+    "marked": "^2.1.3"
```
Comment (christophe-kamphaus-jemmic) [inline package.json]: Trailing whitespace
```suggestion
    "marked": "^2.1.3"
```

**Review by christophe-kamphaus-jemmic — APPROVED** [review]

**Comment by dione-kist-jemmic** [comment]
@christophe-kamphaus-jemmic please, next PR select squash commits on merge button to get history more clean.
image

**Comment by christophe-kamphaus-jemmic** [comment]
I tested it by writing a comment in PR https://github.com/jemmic/task-completed-checker-action/pull/2
https://github.com/jemmic/task-completed-checker-action/actions/runs/6260251186/job/16997932419?pr=2
image

**Pull request merged**